tests/Manager/PageManagerTest.php

```diff
@@ -168,7 +168,9 @@ public function testFindForViewThrowsNotFoundHttpExceptionIfNotFoundInTable()
         {
             $this->expectException('\Symfony\Component\HttpKernel\Exception\NotFoundHttpException');
             $connection = $this->stubConnection();
-            $connection->expects($this->once())->method('fetchColumn')->will($this->returnValue(null));
+            $statement = $this->createMock('Doctrine\DBAL\Driver\Result');
```

Comment:
```suggestion
            $statement = $this->createMock(Result::class);
```
Niet gewoon zo? `::class` triggered verder niets (laadt de class niet in, check zelfs niet of 'ie bestaat)

Reply:
Mja, eigenlijk alle mocks worden op die manier gecreeerd in deze test, dan is het wellicht logischer om de conventies aan te houden die er al zijn. Vandaar denk ik dat ik het zo gedaan heb.